README.md

```diff
@@ -95,6 +95,8 @@ Or, again, if you're compiling and running using Gradle:
 
 `$ gradlew run --args="-cli path/to/config  convert-to-cdf"`
 
+Note: if you convert a source to CDF and that source uses an overvoteLabel or an undeclaredWriteInLabel, you'll need to update each label in your new CDF-based config to match the ContestSelection ID assigned to the label in the generated CDF file.
```

Comment:
Kind of hard for me to follow this as someone who is coming in with uneducated eyes. So this means, if they set overvoteLabel to "overvote" and UWI label to "UWI", they need to manually go through the resulting .json file and Ctrl-R to replace each instance with "overvote-mycontest" and "UWI-mycontest", or what exactly? Could you maybe give an example in the doc here?

Reply:
Also, if this is the case, isn't it something we could do programmatically during the conversion process?

Reply:
No, what I'm saying here is they'll need to set a different label in the _config_ that they use with the new CDF source file. They shouldn't try to modify the generated CDF source file. I'll clarify this text a bit.